src/jvmMain/kotlin/io/github/fourlastor/settings/SettingsViewModel.kt

```diff
@@ -61,4 +69,43 @@ class SettingsViewModel constructor(
     fun angleGles20(angleGles20: Boolean) {
         manager.update { it.angleGles20(angleGles20) }
     }
+
+    override fun runPokeWilds(state: SettingsState.Loaded) {
+        val runArgs = mutableListOf("java", "-jar", state.jar).apply {
+            if (state.angleGles20) {
+                add("angle_gles20")
+            }
+            if (state.devMode) {
+                add("dev")
+            }
+        }.toTypedArray()
+
+        scope.launch(Dispatchers.IO) {
+            val proc = Runtime.getRuntime().exec(
+                runArgs,
+                null,
+                File(state.dir)
+            )
+
+            if (state.devMode) {
```

Comment:
devMode is a very common option for some players (it's more like god mode + editor), so I'd be for using a different field here, something like `state.logsEnabled` maybe?

Reply:
Good idea 👍

Reply:
updated